src/submission-handlers/testOrderHandler.ts

```diff
@@ -0,0 +1,61 @@
+import { map } from 'rxjs/operators';
+import { SubmissionHandler } from '..';
+import { OpenmrsEncounter, OHRIFormField } from '../api/types';
+import { EncounterContext } from '../ohri-form-context';
+
+export const TestOrderHandler: SubmissionHandler = {
+  handleFieldSubmission: (field: OHRIFormField, value: any, context: EncounterContext) => {
+    const constructNewOrder = {
+      action: 'new',
+      urgency: 'ROUTINE',
```

Comment:
The AFE only captures "concept", "type", "orderer" & "careSetting". Do we need ad hoc fields to capture some of these datapoints? I'm not sure whether we should always default to "ROUTINE".